fastai/text/models/awd_lstm.py

```diff
@@ -196,4 +198,72 @@ def awd_lstm_clas_split(model:nn.Module) -> List[nn.Module]:
                           hidden_p=0.1, input_p=0.2, embed_p=0.02, weight_p=0.15, tie_weights=True, out_bias=True)
 
 awd_lstm_clas_config = dict(emb_sz=400, n_hid=1150, n_layers=3, pad_token=1, qrnn=False, bidir=False, output_p=0.4, 
-                       hidden_p=0.2, input_p=0.6, embed_p=0.1, weight_p=0.5)
\ No newline at end of file
+                       hidden_p=0.2, input_p=0.6, embed_p=0.1, weight_p=0.5)
+
```

Comment:
Too many hidden spaces here. Please do the import at the beginning with the other ones.